examples/guides/gpio/Makefile

```
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = hello-world
+
+# Change this to your board if you want to build for a different board
+BOARD ?= feather-nrf52840-sense
```

```suggestion
BOARD ?= arduino-feather-nrf52840-sense
```

The board name `feather-nrf52840-sense` is deprecated and causes a warning message. Not ideal for a tutorial 😅

---

This is also relevant for the other example(s) using the Feather.